ledger-core/idl/wallet_store.djinni

```diff
@@ -0,0 +1,50 @@
+@import "configuration.djinni"
+@import "currency.djinni"
+@import "error.djinni"
+@import "wallet.djinni"
+
+# Class representing a cache of wallets.
+WalletStore = interface +c {
+    # Return number of wallets instanciated under wallet pool.
+    # @param callback, Callback object returns a 32 bits integer, count of wallets
+    getWalletCount(callback: callback2<void, optional<i32>, optional<Error>>);
+
+    # Get instanciated wallets having index in a given range.
+    # @param from, 32 bits integer, lower bound of indices to pick
+    # @param to, 32 bits integer, upper bound of indices to pick
+    # @param callback, ListCallback object returns a list of Wallet objects
+    getWallets(from: i32, size: i32, callback: callback2<void, optional<list<Wallet>>, optional<Error>>);
+
+    # Get wallet with a giver name.
+    # @param name, string, name of wallet to look for
+    # @param callback, Callback object returns a Wallet object
+    getWallet(name: string, callback: callback2<void, optional<Wallet>, optional<Error>>);
+
+    # Update wallet configuration
+    # @param name, string, name of wallet to update
+    # @param configuration, DynamicObject object, configuration object with fields to update
+    # @param callback, Callback object returns the error code, returns ErrorCode::FUTURE_WAS_SUCCESSFULL if everything is fine
+    # > Note: other fields that are not passed in 'configuration' parameter
+    # > that might have been created before remain intact
+    updateWalletConfig(name: string, configuration: DynamicObject, callback: callback2<void, optional<ErrorCode>, optional<Error>>);
+
+    # Instanciate a new wallet under wallet pool.
```

Comment:
Do we have always that notion of wallet pool ? Is it wallet store instead ?

Reply:
Good call, I need to update those comments.